samples/highcharts/studies/stocktools-gui/demo.js

```diff
@@ -1,5 +1,17 @@
 Highcharts.stockChart('container', {
     series: [{
+        // line
+        showPrice: {
+            enabled: true,
+            color: 'red'
+        },
+        // label
+        priceIndicator: {
+            enabled: true,
+            label: {
+                enabled: true
```

Comment:
Can this be set by default? When priceIndicator is enabled, label is enabled too.